javascript/features/player_settings/player_settings_supplement.js

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 Las Venturas Playground. All rights reserved.
+// Use of this source code is governed by the MIT license, a copy of which can
+// be found in the LICENSE file.
+
+import { Supplement } from 'base/supplementable.js';
+import PlayerSetting from 'entities/player_setting.js';
+import { PlayerSettingsDatabase } from 'features/player_settings/player_settings_database.js';
+import SettingList from 'entities/player_setting_list.js';
```

Comment:
Should this move to the new feature in the future?

Reply:
We need this to make the ".settings" supplement work. These are the settings the supplement supports; thus responsibiliy for the supplement in my opinion. 

However; after giving it another thought it might be nice (but maybe slightly overkill) to add a PlayerSettingsManager. This will do the `UpdateSettings` `LoadSettings` and initialize the initial value.

Th